### PR TITLE
PUB-3589 | Add new title field in the composer for Pinterest

### PR DESCRIPTION
**Pull request by MayaUribe**
## Description
Add new title field in the composer for Pinterest under an application mode.

## Context & Notes

- [[PUB-3589](https://buffer.atlassian.net/browse/PUB-3589)] Include new title field for Pinterest
- The new title field is under the depaywall-pinteres-web application mode (disabled at the moment)
- I made changes in the stylings to have a similar look and feel to the links in the IG footer to keep consistency
- The styling changes would be visible for all users

### Best Practices Checklist
-   [x] My code follows the code style and guidelines of this project. 
-   [x] I have added tests to cover my changes.
-   [x] I have tested different user stories.
-   [ ] I have identified similar or related features and tested they are still working.
-   [ ] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [ ] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

### ⚠️ Pre-deploy Checklist

- [ ] I have manually loaded the URL of the staging deployment and confirmed it's working (see below**)
- [ ] I have confirmed that the Cypress E2E test is passing (_Note: we have a known issue where it doesn't run on some PRs. Hopefully this will be fixed soon._)
- [ ] I have confirmed that all other tests are passing.

** To visit the URL of the staging deployment, please vlick the link in the bot comment, or look under checks at the bottom of this PR and click "Details" next to `bufferbotbrains/cicd-buffer-publish`
image

### CODE REVIEW

File: packages/composer/composer/lib/validation/ValidateDraft.js [hunk 1]
```diff
@@ -127,6 +127,19 @@ const validateDraftFunctions = [
     return new ValidationSuccess();
   },
 
+  function maxCharactersInTitle(draft) {
+    if (
+      draft.service.name === 'pinterest' &&
+      draft.service.titleCharLimit !== null &&
+      draft.characterTitleCount > draft.service.titleCharLimit
```
Comment (karelm) [inline packages/composer/composer/lib/validation/ValidateDraft.js:134]: can any of these be null/missing?

Comment (karelm) [reply packages/composer/composer/lib/validation/ValidateDraft.js:134]: e.g. `draft?.service?.name`

Comment (MayaUribe) [reply packages/composer/composer/lib/validation/ValidateDraft.js:134]: Ah good one @karelm! 🙌

**Review by karelm — APPROVED** [review]
LGTM

**Pull request merged**